docs/supported_workflows.md

```diff
@@ -0,0 +1,153 @@
+# Supported Workflows
+
+`pulp-cli` is still in beta, so the features and workflows listed here are subject to change.
+Currently `pulp-cli` supports 5 versions of `pulpcore` from 3.11-3.16 and 5 of Pulp's plugins:
+`pulp_ansible`, `pulp_container`, `pulp_file`, `pulp_python` and `pulp_rpm`. Some of Pulp's other
+plugins can be added to the CLI through CLI plugins, check out [CLI plugins](index) for more information.
+
+## pulpcore
+
+Pulpcore commands require a minimum version of 3.11. Not every command is supported for all
+`pulpcore` versions. Run `pulp status` to check your connection to Pulp and see currently 
+installed plugins.
+
+### Workflows
+
+The CLI currently supports the following workflows:
+* [**Uploading** Artifacts](https://docs.pulpproject.org/pulpcore/workflows/upload-publish.html)
+* [**Creating, Syncing, Distributing** Content](https://docs.pulpproject.org/pulpcore/workflows/exposing-content.html)
+* [**Exporting** Content](https://docs.pulpproject.org/pulpcore/workflows/import-export.html#exporting)
+* [**Labeling** Resources](https://docs.pulpproject.org/pulpcore/workflows/labels.html)
+
+### Features
+
+The CLI currently supports the following operations on these `pulpcore` objects
+(C = Create, R = Read, U = Update, D = Delete):
+
+* Access Policies - RU
+* Artifacts - CR
+* RBAC Content Guards - CRUD, Assign, Remove
+* Exports Pulp - CRD
+* Groups - CRD, Add Permissions, Remove Permissions, Add Users, Remove Users
+* Signing Services - R
+* Tasks - RD, Cancel
+* Users - R
+
+
+## pulp_ansible
+
+Ansible commands require minimum version of 0.7.0. The commands can be found under the `ansible`
+subgroup.
+
+### Workflows
+
+The CLI currently supports the following workflows:
```

Comment:
Same here as above - in fact, all the Workflows sections. I think they need a blank line before the lists?